Dockerfile

```
@@ -1,8 +1,6 @@
-FROM node:8.8.1-alpine
+FROM node:12.9.1-alpine
```

Same as above, do we have to lock this down?

```suggestion
FROM node:12-alpine
```

---

It depends on the use case I guess.
In this case, we can use either as this Dockerfile is only used for testing locally (when running the command `npm run dockerTest`), but in general it is better to specify a particular tag to guarantee the same behaviour between dev/qa/prod.